### PR TITLE
Bump `sequelize` from v5 to v6

### PR DESCRIPTION
**Pull request by lukasaric**
- Upgrading `sequelize` (and `sequlieze-cli`) to latest version
- See [upgrade docs](https://sequelize.org/master/manual/upgrade-to-v6.html) :scroll:

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
 'use strict';
 
+const Promise = require('bluebird');
 const { promisify } = require('util');
-const bluebird = require('bluebird');
-const sequelize = require('sequelize');
 
-if (process.env.NODE_ENV !== 'production') {
-  sequelize.Promise.config({ longStackTraces: true });
```
Comment (lukasaric) [inline server/index.js:8]: `Sequelize.Promise` is deprecated.

File: server/common/database/migrations/20181107134944-unique-program-name.js [hunk 2]
```diff
@@ -4,14 +4,10 @@ const TABLE_NAME = 'program';
...
-  },
-
-  down: (queryInterface, Sequelize) => {
-    return queryInterface.removeConstraint(TABLE_NAME, CONSTRAINT_NAME);
-  }
+  up: qi => qi.addConstraint(TABLE_NAME, {
+    name: CONSTRAINT_NAME,
+    type: 'unique',
+    fields: ['name']
+  }),
```
Comment (lukasaric) [inline server/common/database/migrations/20181107134944-unique-program-name.js:11]: `QueryInterface.addContraint` signature has been updated ([source](https://bit.ly/3gIz183)).

File: server/common/database/helpers.js [hunk 3]
```diff
@@ -1,14 +1,11 @@
...
-const { Sequelize, Op } = require('sequelize');
+const { Op, Sequelize } = require('sequelize');
 const has = require('lodash/has');
 const inRange = require('lodash/inRange');
 const last = require('lodash/last');
 
-// eslint-disable-next-line no-extra-parens
-const AsyncFunction = (async function () {}).constructor;
-
-const isAsyncFunction = arg => arg instanceof AsyncFunction;
```
Comment (lukasaric) [inline server/common/database/helpers.js:11]: Now all methods are wrapped, not only those declared as `async` (Same implementation as in [tailor](https://github.com/ExtensionEngine/tailor)). @vladimyr

**Comment by underscope** [comment]
@lukasaric Please revisit https://github.com/ExtensionEngine/boutique/pull/52. Close upon applying stuff that is still relevant.

**Pull request merged**